.gitignore

```diff
@@ -1,2 +1,3 @@
 /bazel-*
+.idea
```

Comment:
I'd prefer to leave this out - see the explanation here. https://github.com/golang/go/issues/21458

(TBH I should remove the profile.out as well).

Reply:
Ok, reverted